### ISSUE TITLE
What is data, why is sized as it is

### ISSUE THREAD

**Issue by johnramsden** [UBC-ECE-Sasha/PIM-Embedding-Lookup#6]
https://github.com/UBC-ECE-Sasha/PIM-Embedding-Lookup/blob/1830cfbfb4fb21d810d3a6411575b717440c2a62/upmem/src/dpu/emb_dpu_lookup.c#L13

_data_ doesn't appear to be used yet, what is it supposed to represent? Perhaps it's size should also be a constant so that we can easily tell what it represents.

**Comment by nzarif** [comment]
TBH, I don't know. AS far as I remember Abbas has written this code.

**Comment by johnramsden** [comment]
Perhaps it should be removed?

**Comment by nzarif** [comment]
I agree. It seems to be something that was meant to be used for testing purposes.
